reduce.js

```diff
@@ -8,3 +8,20 @@ var numbers = [1, 56, 2, 4, 1, 99, 3, 5];
 // via multiplication
 
 
+//================================================
+ // BONUS ONLY: Look up docs to read about reduce, which takes an array and distills it into a single value.
+//================================================
+
+var numbers = [1, 56, 2, 4, 1, 99, 3, 5];
+
+// Your code here!
+
+// Write something that takes the above array and returns the product of all the numbers
+// via multiplication
+
+
+const newReducer = numbers.reduce((x, y) => {
+  return x * y
+})
+
+console.log(newReducer)
```

Comment:
nice!!! 💯 great job trying this bonus problem, and getting this to work! 

One note about this method: it's typical to name the first parameter `accum` (in your case, the `x` should be `accum`). This is not due to the code breaking if you don't do that, but for human readability; the accumulator is the heart and soul of the `reduce()` method, so it's important to know which of the parameters it is.

The last thing I'll say is just a question for you to think about: if you don't explicitly name the accumulator, how does `reduce` choose the accumulator to start? It's a small but important detail to know when you use this method.